jigna/qt_server.py

```diff
@@ -26,6 +26,9 @@
 
 #: Path to jigna.js file
 JIGNA_JS_FILE = join(abspath(dirname(__file__)), 'js', 'dist', 'jigna.js')
+JIGNA_VUE_JS_FILE = join(
```

Comment:
You probably don't need to include this, right? The vue.js file is already included in `dist/jigna.js`.